hosting/files/control-tower.js

```diff
@@ -0,0 +1,78 @@
+import init from "./base-init.js";
+import {
+    drawProductCardVonRequestURL,
+    drawProductCardVonLocalStorage,
+} from "./product-card-draw.js";
+import {
+    showSearchPage,
+    hideSearchPage,
+    showOnlySaleProduct,
+    hideOnlySaleProduct,
+    showMusinsaOnlyProduct,
+    showAllProductIncludeNotMusinsaOnly,
+    showSoldOutProduct,
+    hideSoldOutProduct,
+} from "./base-header-menu.js";
+import { addFilterTag, removeFilterTag } from "./tagging.js";
+
+export default function ControlTower() {
+    this.controlSwitches = {
+        searchOnOffSwitch: false,

```

Comment:
boolean 타입의 경우 보통 `is~~`로 네이밍합니당.
search라는 게 어떤 걸 가리키는지 모르겠지만 제맘대로 검색창이라고 생각하고 바꿔볼게요!

```suggestion
        isSearchBarOpened: false,
```

Reply:
@hongbiii 검색창 맞습니다! 좋은 정보 알려줘서 고마워요ㅠㅠ